ExtendedColorSchemes/HarmonyPatches/PlayerDataFileManagerSO.cs

```diff
@@ -40,6 +40,14 @@ private static void Postfix()
             try
             {
                 File.Copy(filePath, backupFilePath, true);
+					
+                // Only keep the last 5 days worth of files to stop file spam.
+                var fiveDaysAgo = DateTime.Now.Date.AddDays(-5);
+                var oldFiles = Directory.EnumerateFiles(Application.persistentDataPath, "PlayerData_*", SearchOption.TopDirectoryOnly)
+                    .Where(path => File.GetCreationTime(path) > fiveDaysAgo)
```

Comment:
This actually does the opposite of what you intend it to do.

Reply:
Oops